Flick/Views/ModalViews/AddCollaboratorsModalView.swift

```diff
@@ -102,28 +114,23 @@ class AddCollaboratorModalView: UIView {
 
         NetworkManager.getFriends { [weak self] friends in
             guard let self = self else { return }
+            self.allFriends = friends
             self.friends = friends
             self.setupFriendsView()
         }
     }
 
     private func setupConstraints() {
-        let collaboratorCellHeight = 57
         let collaboratorsTitleLabelSize = CGSize(width: 117, height: 22)
         let copyLinkButtonSize = CGSize(width: 48, height: 12)
         let horizontalPadding = 24
         let inviteTitleLabelSize = CGSize(width: 48, height: 22)
-        let noFriendsSectionViewHeight = 193
+        let noFriendsSectionViewHeight = 3 * collaboratorCellHeight
         let roundButtonSize = CGSize(width: 84, height: 40)
         let verticalPadding = 36
 
-        let collaboratorsTableViewHeight = min(collaborators.count, 4) * collaboratorCellHeight
-        let friendsTableViewHeight = min(friends.count, 4) * collaboratorCellHeight
-
-        let inviteSectionHeight = friends.count > 0 ? friendsTableViewHeight : noFriendsSectionViewHeight
```

Comment:
not checking this anymore bc we didn't get friends yet in the beginning. I think it's ok that we just make the friends part 3 * cell height.